src/libs/utils.js

```diff
@@ -369,12 +369,30 @@ export const truncateInteger = integer => {
   return `${Math.floor(integer / 1000)}k`
 }
 
-// can this be done with usePollingEffect?
-export const poll = async (pollFn, pollTime, pollUntil) => {
-  let result = await pollFn()
-  while (!pollUntil(result)) {
```

Comment:
Thank you for adding this helpful JSDoc below. It just misses `*` on lines between `/**` and `*/`. [Examples here](https://jsdoc.app/about-getting-started.html) in case it helps.